### ISSUE TITLE
Stator: Unknown tag type Hashtag

### ISSUE THREAD

**Issue by kromonos** [jointakahe/takahe#177]
```
takahe-runner-2        |   File "/usr/local/lib/python3.11/site-packages/asgiref/current_thread_executor.py", line 22, in run
takahe-runner-2        |     result = self.fn(*self.args, **self.kwargs)
takahe-runner-2        |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
takahe-runner-2        |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 476, in thread_handler
takahe-runner-2        |     return func(*args, **kwargs)
takahe-runner-2        |            ^^^^^^^^^^^^^^^^^^^^^
takahe-runner-2        |   File "/takahe/activities/models/post.py", line 758, in handle_create_ap
takahe-runner-2        |     cls.by_ap(data["object"], create=True, update=True)
takahe-runner-2        |   File "/takahe/activities/models/post.py", line 679, in by_ap
takahe-runner-2        |     raise ValueError(f"Unknown tag type {tag['type']}")
takahe-runner-2        | ValueError: Unknown tag type Hashtag
```

**Comment by andrewgodwin** [comment]
Right! Because I implemented the schema changes! Fix incoming.

**Comment by andrewgodwin** [comment]
Should be fixed in 45c6978
